.cbmc-batch/jobs/memset_override_0/Makefile

```diff
@@ -0,0 +1,29 @@
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may not use
+# this file except in compliance with the License. A copy of the License is
+# located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+# implied. See the License for the specific language governing permissions and
+# limitations under the License.
+
+###########
+#NOTE: If we don't use the unwindset, leave it empty
+UNWINDSET += memset_impl.0:161
+UNWINDSET += memset_override_0_impl.0:21
```

Comment:
why 161 or 21?

Reply:
I can make that Makefile math and add a comment